src/pudl/metadata/fields.py

```diff
@@ -895,6 +938,28 @@
         "description": "Gross megawatt-hours delivered in power exchanges and used as the basis for settlement.",
         "unit": "MWh",
     },
+    "demand_adjusted_mwh": {
+        "type": "number",
+        "description": "Electricity demand adjusted by EIA to reflect non-physical commercial transfers through pseudo-ties and dynamic scheduling.",
+        "unit": "MWh",
+    },
+    # TODO[zaneselvans] 2024-04-20: Is the timestamp when the forecast was made? Or the
+    # time at which the forecast is trying to predict demand?
```

Comment:
just noting this remaining todo (which seems truly still undone)

Reply:
Yep, still out there. Will post Q more broadly.